docs/extending-restier/additional-operations.md

```diff
@@ -0,0 +1,69 @@
+## Additional WebAPI Operations
+
+RESTier is built on top of ASP.NET Web API, so like our regular OData support, augmenting your service
+with additional actions is very simple.
+
+First, you must add the action to the EDM Model Builder.
+
```

Comment:
This need some update, user can define a method in Api class with operation attribute, then RESTier will auto build the model for end user or end user can use customized model builder to build function by himselves. We recommend the first way as we are working to get rid of additional controller now, a PR for function is already in github.